src/Courses/TopicSettings/ProblemSettings.tsx

```diff
@@ -117,7 +118,10 @@ export const ProblemSettings: React.FC<ProblemSettingsProps> = ({selected, setSe
             const newTopic = new TopicObject(topic);
             const newQuestion = _.find(newTopic.questions, ['id', selected.id]);
 
+            // TODO: Right now, the backend does not return the courseQuestionAssessmentInfo, so we should use the local data
+            // if the attempt was a success.
```

Comment:
I thought we found that courseQuestionAssessmentInfo _was_ being returned by the backend, no?

Reply:
It was being returned from the GET call, but it's not returned from the PUT call (because of some finnickiness with the upsert operation)